src/manage_nsfs/manage_nsfs_cli_errors.js

```diff
@@ -16,18 +18,21 @@ class ManageCLIError extends Error {
     /**
      * @param {ManageCLIErrorSpec} error_spec 
      */
-    constructor({ code, message, http_code }) {
-        super(message); // sets this.message
+    constructor({ code, message, http_code, detail, cause }) {
+        super(message, { cause });
         this.code = code;
         this.http_code = http_code;
+        this.detail = detail;
+        this.cause = cause;
```

Comment:
No need to set this.cause because the standard Error ctor which is called in super() will do that.